### PR TITLE
[MU4] fix #279780 Better selection of orchestra instruments

### PR DESCRIPTION
**Pull request by Laurelin**
Replaced Baritone Oboe with Bass Oboe - needs to be differentiated via translation. Bass Oboe is English, Spanish, Italian. Baritone is German, French.

Deleted Contrabass Cl, tuba keys, as they are never used in orchestra music. Added many keys of trumpet and horn, C clarinet, a cornet, Cimbasso, Ophelcide, oboe d'amor, bass oboe, tenor trombone, some unpitched percussion to orchestra genre group

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
Continued from #3476

File: share/instruments/instruments.xml [hunk 1]
```diff
@@ -1001,6 +1001,7 @@
                   <Channel>
                         <program value="68"/>
                   </Channel>
+				  <genre>orchestra</genre>
```
Comment (anatoly-os) [inline share/instruments/instruments.xml]: Something is wrong with mixing whitespaces/tabs. We prefer whitespaces everywhere.

File: share/instruments/instruments.xml [hunk 2]
```diff
@@ -1035,18 +1036,22 @@
...
+                  <shortName>B. Ob.</shortName>
+                  <description>Bass Oboe</description>
                   <musicXMLid>wind.reed.oboe.bass</musicXMLid>
-                  <clef>G8vb</clef>
+                  <transposingClef>G</transposingClef>
+                  <concertClef>G8vb</concertClef>
                   <barlineSpan>1</barlineSpan>
                   <aPitchRange>47-76</aPitchRange>
                   <pPitchRange>47-78</pPitchRange>
+				  <transposeDiatonic>-7</transposeDiatonic>
```
Comment (anatoly-os) [inline share/instruments/instruments.xml]: Whitespaces/tabs

File: share/instruments/instruments.xml [hunk 3]
```diff
@@ -1035,18 +1036,22 @@
...
+                  <transposeChromatic>-12</transposeChromatic>
                   <Channel>
                         <program value="68"/>
                   </Channel>
+				  <genre>orchestra</genre>
```
Comment (anatoly-os) [inline share/instruments/instruments.xml]: Whitespaces/tabs

File: share/instruments/instruments.xml [hunk 4]
```diff
@@ -1671,6 +1676,7 @@
                   <Channel>
                         <program value="71"/>
                   </Channel>
+				  <genre>orchestra</genre>
```
Comment (anatoly-os) [inline share/instruments/instruments.xml]: Whitespaces/tabs

File: share/instruments/instruments.xml (hunk 1 above)
Comment (Jojo-Schmitz) [reply share/instruments/instruments.xml]: Tabs are whitespace ;-). We prefer spaces (which are also whitespace)

**Comment by anatoly-os** [comment]
@Laurelin hi! Are you around to address the comments from the review? Let me know whether you have time, otherwise I'll manually apply the changes to master.

**Comment by Jojo-Schmitz** [comment]
The Bariton/Bass Oboe issue is not resloved

**Comment by dmitrio95** [comment]
Probably needs a rebase

**Comment by Jojo-Schmitz** [comment]
There's nothing left for this PR to do but to add/remove some genre lines, the rest has been done quite a while ago already

**Comment by dmitrio95** [comment]
So does this part need to be redone, or perhaps in a new PR?

**Comment by Jojo-Schmitz** [comment]
Well, rebase should fix this, I think. We'd see once that is done ;-)

**Comment by vpereverzev** [comment]
Archived due to inactivity for a long time

**Pull request closed without merging**